middleware/formatHotelPost.js

```diff
@@ -0,0 +1,10 @@
+const formatHotelPost = (req, res, next) => {
+  const hotel = req.body;
+  hotel.name = hotel.name
+    .toLowerCase()
+    .replace(/\b([a-z])/gi, char => char.toUpperCase());
+  req.body = { ...hotel };
+  next();
+};
+
+module.exports = formatHotelPost;
```

Comment:
Isn't this module doing the same thing as `middleware/formatHotelChange.js` ?

Reply:
The slight diff was that formatHotelChange first had a check to see whether req.body included a name change. As this conditional will pass every time for formatHotelPost, I just combined the two together.